libs/verify-ios-ci-environment.sh

```diff
@@ -37,14 +32,6 @@ if ! [[ -x "$(command -v xcpretty)" ]]; then
   exit 1
 fi
 
-# For Xcode 12 make sure EXCLUDED_ARCHS is set to arm architectures otherwise
```

Comment:
I'm assuming this is no longer needed with Xcode 13 or was this a "carthage made us do it, and now we don't need it"?

Reply:
Yeah and this makes it impossible to build natively on M1 due to arm64 being excluded as a simulator option